dotnet-ory-network/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM mcr.microsoft.com/devcontainers/dotnet:0-7.0
+RUN curl https://raw.githubusercontent.com/ory/meta/master/install.sh | bash -s -- -b /usr/local/bin/ ory
```

Comment:
I've seen a couple of missing newlines at EOF. Would you mind configuring your editor to add them on save? 👍